### PR TITLE
Use fuzzy matching for looking up forks/followers in pg:info

### PR DESCRIPTION
**Pull request by msakrejda**
See https://github.com/heroku/shogun/pull/7712

### CODE REVIEW

**Comment by msakrejda** [comment]
@camilleldn is there anything in the credentials world that this change would cause problems for?

File: lib/util.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const debug = require('./debug')
 const getBastion = require('./bastion').getBastion
 const sortBy = require('lodash.sortby')
 const printf = require('printf')
+const URL = require('url').URL
```
Comment (jdx) [inline lib/util.js:8]: TIL url.parse is legacy

**Comment by jdx** [comment]
code lgtm aside from test failure

**Review by jdx — APPROVED** [review]

**Comment by msakrejda** [comment]
Hmm, I'm not sure what's up with the failure: tests pass locally. Investigating.

**Comment by msakrejda** [comment]
The issue was due to 6.2.2 behaving differently. I did not dig in since the CLI is on a more recent version.

**Review by camillebaldock — APPROVED** [review]
👍 this will work well with credentials

**Comment by jdx** [comment]
I think that `new URL` syntax is new

**Pull request merged**